test/clean_log.test.ts

```
@@ -89,93 +95,93 @@ describe('test/clean_log.test.js', () => {
-    assert(fs.existsSync(path.join(logDir, filepath)) === false);
+    assert.equal(fs.existsSync(path.join(logDir, filepath)), false);
   });
 
-  it('should error when readdir err', function* () {
-    mm(require('mz/fs'), 'readdir', dir => new Promise((_, reject) => reject(new Error(`Permission: readdir ${dir}`))));
-    let message;
-    mm(app.coreLogger, 'error', err => (message = err.message));
+  it('should error when readdir err', async () => {
+    mm(fsPromises, 'readdir', (dir: string) => new Promise((_, reject) => reject(new Error(`Permission: readdir ${dir}`))));
+    let message = '';
+    mm(app.coreLogger, 'error', (err: Error) => (message = err.message));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)